### PR TITLE
Refactor wishlists_controller_spec.rb

### LINKED ISSUE #64: Refactor specs of the WishlistsController
~~Use gem Faker in factories and add gem database cleaner.~~

Reference

### PR DESCRIPTION
**Pull request by nodzy**
Closes #64

### CODE REVIEW

File: spec/controllers/wishlists_controller_spec.rb [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+    it "redirects to the 'sign_in' page" do
+      expect(response).to redirect_to(new_user_session_path)
+    end
+  end
+
+  shared_examples "redirects to the wishlists index page" do
+    it "redirects to the wishlists index page" do
+      expect(response).to redirect_to(wishlists_path)
+    end
+  end
```
Comment (student-tomasz) [inline spec/controllers/wishlists_controller_spec.rb:16]: You could make this a one `shared_example` like so:

``` ruby
shared_examples "redirects to" do |path_helper|
  it "redirects to the #{path_helper}" do
    expect(response).to redirect_to(send(path_helper))
  end
end
```

i wywoływać to później w  ten sposób:

``` ruby
include_examples "redirects to", :new_user_session_path
```

Comment (nodzy) [inline spec/controllers/wishlists_controller_spec.rb:16]: Good idea, thank you. I refactored it

File: spec/controllers/wishlists_controller_spec.rb [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+    it "redirects to the #{path_helper}" do
+      expect(response).to redirect_to(send(path_helper))
+    end
+  end
+
+  shared_examples "shows a flash message" do |flash_type|
+    it "shows a/an #{flash_type}" do
+      expect(flash[flash_type.to_s]).not_to be_nil
+    end
+  end
```
Comment (student-tomasz) [inline spec/controllers/wishlists_controller_spec.rb:16]: I've commented on the exact same thing here:
https://github.com/Ragnarson-Internship2016/group-I/pull/86/files#r72648899

If you think it's helpful, then implement this.

**Comment by student-tomasz** [comment]
👍

**Comment by agalkiewicz** [comment]
:+1:

**Comment by gotar** [comment]
squash and you can merge

**Pull request merged**